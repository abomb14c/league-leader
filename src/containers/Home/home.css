.home-container{
  display: flex;
  width: 100%;
}

.home-title {
  color: #FDF3E7;
  font-weight: 200;
}
