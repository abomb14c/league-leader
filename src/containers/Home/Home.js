import React from 'react';
import EPLContainer from '../EPLContainer/EPLContainer';
import NBAContainer from '../NBAContainer/NBAContainer';
import { Link } from 'react-router-dom';
import './home.css';

export const Home = (props) => {

  return (
    <div className="home-container">
      <h1 className="home-title">Current Standings</h1>
      <p><Link to="/setupleague"
        className="setup-link"
      >
      Start A New League
      </Link></p>
      <div className="standings-container">
        <EPLContainer/>
        <NBAContainer />
      </div>
    </div>
  );
};

export default Home;