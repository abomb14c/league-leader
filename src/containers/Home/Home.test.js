import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import  { Home }  from './Home';

describe('Home', () => {
  let wrapper;

  beforeEach(() => {

    wrapper = shallow(<Home/>);
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});