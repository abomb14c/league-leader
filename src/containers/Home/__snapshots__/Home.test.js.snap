// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1
          className="home-title"
>
          Current Standings
</h1>,
        <p>
          <Link
                    className="setup-link"
                    replace={false}
                    to="/setupleague"
          >
                    Start A New League
          </Link>
</p>,
        <div
          className="standings-container"
>
          <Connect(Component) />
          <Connect(Component) />
</div>,
      ],
      "className": "home-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Current Standings",
          "className": "home-title",
        },
        "ref": null,
        "rendered": "Current Standings",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <Link
            className="setup-link"
            replace={false}
            to="/setupleague"
>
            Start A New League
</Link>,
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Start A New League",
            "className": "setup-link",
            "replace": false,
            "to": "/setupleague",
          },
          "ref": null,
          "rendered": "Start A New League",
          "type": [Function],
        },
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Connect(Component) />,
            <Connect(Component) />,
          ],
          "className": "standings-container",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1
            className="home-title"
>
            Current Standings
</h1>,
          <p>
            <Link
                        className="setup-link"
                        replace={false}
                        to="/setupleague"
            >
                        Start A New League
            </Link>
</p>,
          <div
            className="standings-container"
>
            <Connect(Component) />
            <Connect(Component) />
</div>,
        ],
        "className": "home-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Current Standings",
            "className": "home-title",
          },
          "ref": null,
          "rendered": "Current Standings",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": <Link
              className="setup-link"
              replace={false}
              to="/setupleague"
>
              Start A New League
</Link>,
          },
          "ref": null,
          "rendered": Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Start A New League",
              "className": "setup-link",
              "replace": false,
              "to": "/setupleague",
            },
            "ref": null,
            "rendered": "Start A New League",
            "type": [Function],
          },
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <Connect(Component) />,
              <Connect(Component) />,
            ],
            "className": "standings-container",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
